### PR TITLE
Language update for filter out unwanted apis

### PR DESCRIPTION
**Pull request by peterennis**

### CODE REVIEW

File: Documentation/tutorial/howto_filter_out_unwanted_apis.md [hunk 1]
```diff
@@ -40,19 +40,19 @@ The path of the configuration file could be specified by the following two ways.
 The format of the filter configuration file
 -------------------------------------------
 
-You could specify `apiRules` with a list of `exclude` or `include` rules.
+Specify `apiRules` with a list of `exclude` or `include` rules.
```
Comment (superyyrrzz) [inline Documentation/tutorial/howto_filter_out_unwanted_apis.md]: In main body, is it more appropriate to use full sentence instead of a phrase?

Comment (peterennis) [inline Documentation/tutorial/howto_filter_out_unwanted_apis.md]: You may specify `apiRules` with a list of `exclude` or `include` rules.
Or
`apiRules` may be specified with a list of `exclude` or `include` rules.
Or
You could specify `apiRules` with a list of `exclude` or `include` rules. 
Or
Specify `apiRules` with a list of `exclude` or `include` rules.
Or
Specify `apiRules` by providing a list of `exclude` or `include` rules. 

Phrases are more common for bulleted lists.
Full sentences are more formal.

Comment (superyyrrzz) [inline Documentation/tutorial/howto_filter_out_unwanted_apis.md]: Yes, I agree. Personally I prefer the 3rd one using `could`.

**Comment by superyyrrzz** [comment]
:+1:

**Comment by ansyral** [comment]
:+1:

**Comment by vwxyzh** [comment]
:+1:

**Comment by qinezh** [comment]
:+1:

**Pull request merged**